apps/exam/models.py

```diff
@@ -34,24 +45,25 @@ class ExamAttempt(models.Model):
   STATUS_FINISHED = 'finished'
   STATUS_IN_PROGRESS = 'in_progress'
 
-  STATUSES = ((STATUS_FINISHED, 'Finished'), (STATUS_IN_PROGRESS,
-                                              'In progress'))
+  STATUSES = ((STATUS_FINISHED, _('Finished')), (STATUS_IN_PROGRESS,
+                                              _('In progress')))
 
   PRACTICE_MODE = 'practice'
   EXAM_MODE = 'exam'
 
   EXAM_MODES = [
-      (PRACTICE_MODE, 'Practice mode'),
-      (EXAM_MODE, 'Exam mode'),
+
+      (PRACTICE_MODE, _('Practice mode')),
+      (EXAM_MODE, _('Exam mode')),
   ]
 
-  created = models.DateTimeField(default=timezone.now, editable=False)
-  duration_minutes = models.IntegerField(default=120)
-  grade = models.IntegerField(default=0)
-  mode = models.CharField(max_length=30,
+  created = models.DateTimeField(verbose_name=_('created'), default=timezone.now, editable=False)
+  duration_minutes = models.IntegerField(verbose_name=_('duration'), default=120)
+  grade = models.IntegerField(verbose_name=_('grade'), default=0)
+  mode = models.CharField(verbose_name=_('mode'), max_length=30,
                           default=PRACTICE_MODE,
                           choices=EXAM_MODES)
-  status = models.CharField(max_length=25,
+  status = models.CharField(verbose_name=_('status'), max_length=25,
                             default=STATUS_IN_PROGRESS,
                             choices=STATUSES)
   user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
Do you want to cover FK and M2M fields as well?

Reply:
As I understood, translations for FK and M2M will be available from those models, as long as there are verbose names in Meta. class So there's no need to add it again. Is that not correct?